jax/experimental/jax2tf/primitives_with_limited_support.md

```diff
@@ -1,5 +1,15 @@
 # Primitives with limited support
 
+*Last generated on (YYYY-MM-DD): 2020-09-08*
+
+## Updating the documentation
+
+This file is automatically generated by running the jax2tf tests in
+`jax/experimental/jax2tf/tests/primitives_test.py` with the
+`JAX2TF_CATEGORIZE_OUT` environment variable set.
+
+## Generated summary of primitives with limited support
+
```

Comment:
I suggest adding some text here: 

The following JAX primitives are converted to Tensorflow but the result of the conversion may trigger runtime errors when run on certain devices and with certain data types.